gcloud-java-storage/src/main/java/com/google/gcloud/storage/testing/RemoteGcsHelper.java

```diff
@@ -89,6 +89,23 @@ public static Boolean forceDelete(Storage storage, String bucket, long timeout,
   }
 
   /**
+   * Deletes a bucket, even if non-empty. Objects in the bucket are listed and deleted until bucket
+   * deletion succeeds. This method can be used to delete buckets from within App Engine.  Note that
+   * this method does not set a timeout.
+   *
+   * @param storage the storage service to be used to issue requests
+   * @param bucket the bucket to be deleted
+   * @throws StorageException if an exception is encountered during bucket deletion
+   */
+  public static void forceDelete(Storage storage, String bucket) throws StorageException {
+    try {
+      new DeleteBucketTask(storage, bucket).call();
+    } catch (Exception e) {
+      throw (StorageException) e;
```

Comment:
This could trigger a `ClassCastException`.

We should either change `DeleteBucketTask.call` method signature and remove the `throws Exception` (not needed as StorageException is a runtime exception if we also change `Thread.sleep` to use guava's [`Uninterruptibles.sleepUninterruptibly`](http://docs.guava-libraries.googlecode.com/git/javadoc/src-html/com/google/common/util/concurrent/Uninterruptibles.html#line.267)) or use
guava's `Throwables.propagateIfInstanceOf(e, StorageException.class)` (and a line
after that add `throw Throwables.propagate(e)`)